packages/http-multipart-formdata/http-multipart-formdata.1.1.0/opam

```diff
@@ -0,0 +1,41 @@
+opam-version: "2.0"
+synopsis: "Http multipart/formdata parser"
+description:
+  "OCaml implementation of RFC 7578 (Returning Values from Forms: multipart/form-data)- https://tools.ietf.org/html/rfc7578"
+maintainer: ["Bikal Lem"]
+authors: ["Bikal Lem, <gbikal@gmail.com>"]
+license: "MPL-2.0"
+tags: ["http" "multipart" "formadata" "form" "web"]
+homepage: "https://github.com/lemaetech/http-mutlipart-formdata"
+bug-reports: "https://github.com/lemaetech/http-mutlipart-formdata/issues"
+depends: [
+  "dune" {>= "2.7"}
+  "ocaml" {>= "4.10.0"}
+  "fmt" {>= "0.8.9"}
+  "reparse" {= "2.1.0"}
```

Comment:
```suggestion
  "reparse" {>= "2.1.0"}
```
Is there a reason this is so strict? We can add an upper bound to the metadata if newer versions of reparse break it in the future, but you could benefit from bugfixes in the meantime.

Reply:
Sure. Makes sense.